packages/syft/src/syft/ast/klass.py

```diff
@@ -839,15 +926,16 @@ def pointerize_args_and_kwargs(
     for arg in args:
         # check if its already a pointer
         if not isinstance(arg, Pointer):
-            arg_ptr = arg.send(client, pointable=False)
+            arg_ptr = arg.send(client, pointable=not gc_enabled)
```

Comment:
@tudorcebere @madhavajay I added here not to garbage collection because sometimes it was triggered by the worker (probably because working with pointer more than usually)

Reply:
does this invert the bool? Should it possibly just be pointable=False incase the bool is already False?

Reply:
I am not sure if I understand what you mean.
Here I only want to make sure that we set `pointable` accordingly to the `gc_enabled` parameter. If we don't want to garbage collect the parameter we should set `pointable to False` (which is the invert of the `gc_enabled` variable)